HomeWork/Program.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System.Numerics;
+using System.Security.Cryptography;
+
+Console.WriteLine("Enter the number of array elements");
+string size = Console.ReadLine();
+
+Console.WriteLine();
+
+int[] elements = new int[Convert.ToInt32(size)];
+
+
+
+for (int i = 0; i < Convert.ToInt32(size); i++)
+{
+    elements[i] = new Random().Next(1, 26);
+}
+
+
+int[] even = new int[Convert.ToInt32(size)];
+int[] uneven = new int[Convert.ToInt32(size)]; 
+
+
+for (int i = 0; i < Convert.ToInt32(size); i++)
+{
+    if (elements[i] % 2 == 0)
+    {
+        even[i] = elements[i];
+    }
+    else
+    {
+        uneven[i] = elements[i];
+    }
+
+}
+
+
+char[] alphabet = "abcdefghijklmnopqrstuvwxyz".ToCharArray();
+
+int evenTotal = 0;
+int unevenTotal = 0;
+
+for (int i = 0; i < Convert.ToInt32(size); i++)
+{
+    if (even[i] != 0)
+    {
+        evenTotal++;
+    }
+    if (uneven[i] != 0)
+    {
+        unevenTotal++;
+    }
+}
+
+Console.WriteLine("An array that contains more uppercase letters:");
+
+if (evenTotal > unevenTotal)
```

Comment:
For what reason we need this? It seems that you try to find whether we have more even or odd numbers, but we need to find whether we have more Upper chars